src/drivers/opengles/lv_opengles_private.h

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file lv_opengles_private.h
+ *
+ */
+
+#ifndef LV_OPENGLES_PRIVATE_H
+#define LV_OPENGLES_PRIVATE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "../../lv_conf_internal.h"
+#if LV_USE_OPENGLES
+
+#if LV_USE_OPENGLES_API == LV_OPENGLES_API_GLFW
+#include <GL/glew.h>
+#include <GLFW/glfw3.h>
+#elif LV_USE_OPENGLES_API == LV_OPENGLES_API_EGL
+#include <GL/gl.h>
+#include <EGL/egl.h>
+#include <GLES3/gl3.h>
+#endif
```

Comment:
Nit: It would be good to have an else here that raises an error, it may help the user during development to tell him : "hey your configuration might be invalid"